src/System.Private.CoreLib/shared/System/ReadOnlySpan.Fast.cs

```diff
@@ -154,6 +154,37 @@ internal ReadOnlySpan(ref T ptr, int length)
 #endif
         }
 
+        public ref readonly T this[Index index]
+        {
+#if PROJECTN
+            [BoundsChecking]
```

Comment:
This will likely cause PROJECTN toolchain to crash.  `BoundsChecking` methods are expected to have `int` argument.